src/FieldError.php

```
@@ -69,52 +69,90 @@ public function clear(string|null $property = null): void
      *
-     * @param string $property Property name.
+     * @param string $field Field name.
      * @param bool $first Whether to return only the first error.
      *
-     * @return array|string Error list for the property, or the first error when `$first` is `true`.
+     * @return array|string Error list for the field, or the first error when `$first` is `true`.
      *
      * @phpstan-return array<int, string>|string
      */
-    public function getProperty(string $property, bool $first = false): array|string
+    public function getField(string $field, bool $first = false): array|string
```

PHPMD (2.15.0)

[error] The method getField has a boolean flag argument $first, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)